src/components/Dashboard/FirstSteps/reducers/firstStepsReducer.js

```diff
@@ -1,7 +1,11 @@
+import { mapSystemUsageSummary } from '..';
+
 export const PENDING_STATUS = 0;
 export const COMPLETED_STATUS = 1;
 export const WARNING_STATUS = 2;
+export const UNKNOWN_STATUS = 3;
```

Comment:
I cannot find where you are filtering the steps with unknown status to avoid rendering them.

Reply:
you're right, it's missing. I already add it

Reply:
Added!